server/src/test/scala/services/crunch/VoyageManifestsSpec.scala

```diff
@@ -70,16 +72,17 @@ class VoyageManifestsSpec extends CrunchTestLike {
     success
   }
 
-  "Given a VoyageManifest and its arrival where the arrival has a different number of passengers to the manifest " >> {
+  "Given a VoyageManifest and its arrival where the arrival has a different number of passengers to the manifest but is within 5%" >> {
```

Comment:
Is the 5% threshold relevant in this test? I notice that we've switch to an ACL flight rather than a live flight